marmoset/installstatus/__init__.py

```diff
@@ -51,8 +51,4 @@ def get_stats(args):
 
 def is_uuid(uuid):
     """Check if provided string is a UUID"""
-    if validation.is_uuid(uuid):
-        return True
-    else:
-        print('Invalid uuid %s.' % uuid)
-        return False
+    return True if validation.is_uuid(uuid) else False
```

Comment:
This seems a bit redundant. Shouldn't this work just as well?

```
return validation.is_uuid(uuid)
```

Reply:
thanks, fixed it